src/NuGet/PackHelper.csproj

```diff
@@ -0,0 +1,260 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard1.0</TargetFramework>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <NuspecProperties>licenseUrl=http://go.microsoft.com/fwlink/?LinkId=529443</NuspecProperties>
+    <NuspecProperties>$(NuspecProperties);version=$(NUGET_PACKAGE_VERSION)</NuspecProperties>
+    <NuspecProperties>$(NuspecProperties);authors=Microsoft</NuspecProperties>
+    <NuspecProperties>$(NuspecProperties);projectURL=http://msdn.com/roslyn</NuspecProperties>
+    <NuspecProperties>$(NuspecProperties);tags=Roslyn CodeAnalysis Compiler CSharp VB VisualBasic Parser Scanner Lexer Emit CodeGeneration Metadata IL Compilation Scripting Syntax Semantics</NuspecProperties>
+    <NuspecProperties>$(NuspecProperties);emptyDirPath=$(MSBuildThisFileDirectory)../NuGet</NuspecProperties>
+    <NuspecProperties>$(NuspecProperties);additionalFilesPath=$(MSBuildThisFileDirectory)../../build/NuGetAdditionalFiles</NuspecProperties>
+    <NuspecProperties>$(NuspecProperties);commitPathMessage=Commit $(COMMIT_SHA)</NuspecProperties>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <NuspecProperties>$(NuspecProperties);BasicUndoVersion=0.9.3</NuspecProperties>
```

Comment:
Is having all these dependency versions hard-coded like this right?  How easy will these update?

Reply:
In the long term def not right. Cant do much until we have the auto-flow implemented. Once that is done though this should be very fixable